addOns/scripts/src/main/java/org/zaproxy/zap/extension/scripts/ConsolePanel.java

```diff
@@ -85,6 +85,9 @@
                 Constant.messages.getString("scripts.changed.replace")
             };
 
+    private Map<ScriptWrapper, Integer> scriptWrapperToOffset =
```

Comment:
We should remove the script wrappers from this map when they are removed, to avoid memory leaks.